ubuntudesign/cloud-init.io
# Add video lightbox to "Watch tutorial" link in hero

## Done
- Updated npm deps
- Added JS util so that "View tutorial" opens in lightbox instead of going through to YouTube 

## QA
- Pull code
- Run gulp
- Click "Watch tutorial" in hero
- Confirm that lightbox appears

## Issue / Card
Fixes: #42
